database/index.js

```diff
@@ -0,0 +1,23 @@
+var mysql = require('mysql');
+
+connection = mysql.createPool({
+  connectionLimit: 100,
+  host: '127.0.0.1',
+  port: 3306,
+  user: 'root',
+  password: '',
+  database: 'related_products'
+});
+
+const getAllProducts = (callback) => {
+  const queryString = 'SELECT * FROM products';
+  connection.query(queryString, (err, res) => {
+    if (err) {
+      callback(err);
+    } else {
+      callback(null, err);
+    }
+  })
+};
+
+module.exports = {getAllProducts}
```

Comment:
Always put a newline at the end of ALL of your files! Installing a linter will tell you this. It will also tell you about your missing semicolons to that you can have your tooling inform you of these issues (rather than relying on your PR reviewers to do so).

Reply:
Thanks Robin, I really appreciate your comments. ESllint is now up and running.